mne/commands/mne_kit2fiff.py

```diff
@@ -40,9 +40,15 @@ def run():
     parser.add_option('--output', dest='out_fname',
                       help='Name of the resulting fiff file',
                       metavar='filename')
+    parser.add_option('--debug', dest='debug', action='store_true',
+                      default=False,
+                      help='Set logging level for terminal output to debug')
 
     options, args = parser.parse_args()
 
+    if options.debug:
+        mne.set_log_level('debug')
```

Comment:
this will make a permanent change in the config file. You should revert the change after running